templates/al2023.Dockerfile.template

```diff
@@ -0,0 +1,49 @@
+FROM amazonlinux:2023
+{% if version_8 %}
+ARG version={{JDK_VERSION}}
+{% else %}
+ARG version={{CORRETTO_VERSION}}
+{% endif %}
+ARG package_version=1
+
+RUN set -eux \
+    {%if version_8 %}
+    && export resouce_version=$(echo $version | tr '-' '.' | tr '_' '.'| tr -d "b" | awk -F. '{print $2"."$4"."$5"."$6}') \
+    {% endif %}
+    && rpm --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-amazon-linux-2023 \
+    && echo "localpkg_gpgcheck=1" >> /etc/dnf/dnf.conf \
+    && CORRETO_TEMP=$(mktemp -d) \
+    && pushd ${CORRETO_TEMP} \
+    {% if jre %}
+    && RPM_LIST=("java-{{JDK_VERSION}}-amazon-corretto-$version.amzn2.$(uname -m).rpm") \
```

Comment:
I think it would simplify the code if you eliminate JDK_VERSION and just set MAJOR_VERSION to 1.8.0 when its 8. Is that possible or are those values too overloaded?

Reply:
I can try that and see if everything builds correctly